flink-runtime-web/src/main/resources/META-INF/NOTICE

```diff
@@ -1,458 +1,2111 @@
-flink-runtime-web
-Copyright 2014-2024 The Apache Software Foundation
-
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
```

Comment:
Can you preserve this header in the NOTICE file (it is common in all Flink modules)?

Reply:
Sorry for this. I've restored this header in NOTICE file and updated the template to handle it.